### ISSUE TITLE
Getting NPM warn for the latest Angular project.

### ISSUE THREAD

**Issue by arunkumar30** [angular/angular-cli#26255]
### Command

version

### Is this a regression?

- [x] Yes, this behavior used to work in the previous version

### Description

npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
npm WARN deprecated @babel/plugin-proposal-unicode-property-regex@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-unicode-property-regex instead.
npm WARN deprecated @babel/plugin-proposal-async-generator-functions@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.
npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm WARN deprecated @wessberg/ts-evaluator@0.0.27: this package has been renamed to ts-evaluator. Please install ts-evaluator instead

### Minimal Reproduction

Adding angular devkit globally also throws the same error.
npm i -g @angular-devkit/build-angular@16.2.9

### Your Environment

```text
_                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 16.2.0
Node: 18.18.0
Package Manager: npm 10.2.3
OS: win32 x64

Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1602.0 (cli-only)
@angular-devkit/core         16.2.0 (cli-only)
@angular-devkit/schematics   16.2.0 (cli-only)
@schematics/angular          16.2.0 (cli-only)
```

**Comment by arunkumar30** [comment]
Attached new angular repro.
[repro-app.zip](https://github.com/angular/angular-cli/files/13278075/repro-app.zip)

on npm install getting below warnings.
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
npm WARN deprecated @babel/plugin-proposal-unicode-property-regex@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-unicode-property-regex instead.
npm WARN deprecated @babel/plugin-proposal-async-generator-functions@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.
npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm WARN deprecated @wessberg/ts-evaluator@0.0.27: this package has been renamed to ts-evaluator. Please install ts-evaluator instead

added 1009 packages, and audited 1010 packages in 15s

**Comment by alan-agius4** [comment]
This has already been addressed via https://github.com/angular/angular-cli/pull/25820 which will be available as part of version 17.0.0 that will be released later this week. 

For version 16 users, these warnings are safe to ignore.
